SPECS/yaml-cpp/yaml-cpp.spec

```
@@ -31,7 +30,7 @@ Requires:       %{name}-devel%{?_isa} = %{version}-%{release}
 The %{name}-static package contains the static library for %{name}.
 
 %prep
-%autosetup -n %{name}-%{name}-%{version}
+%autosetup -n %{name}-%{version}
```

If the folder name is now `%{name}-%{version}`, then that's the default and we can simplify:
```suggestion
%autosetup
```